travis_configure.sh

```diff
@@ -35,6 +35,11 @@ before_install()
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
             gem install fpm
+            if [[ -n "$TRAVIS_TAG" ]]; then
```

Comment:
Could you add `jprog` to the dependency which is installed in the doc. Something like

```
# Installs dependencies
# linux:
#   deb:
#     - fpm
#     - jfrog (If the current build is for a git tag)
```